src/foam/nanos/notification/email/EmailTemplate.js

```diff
@@ -26,7 +30,7 @@ foam.CLASS({
 
   properties: [
     {
-      class: 'Long',
+      class: 'String',
```

Comment:
Also need to update the nspec for EmailTemplate, changing from setSeqNo to setGuid.